### ISSUE TITLE
Solve / Debug reason for empty outputs

### ISSUE THREAD

**Issue by erinCodes3276** [queens-supercluster/PREWL#22]
Output / prediction is returning empty occasionally from unrelated new_inputs. The only pattern I have found so far is that output is empty when new_input is word for word one of the prompt examples. Other than that case, I can't find any relation with the other times.

Example where it fails: 
https://github.com/queens-supercluster/conversation_analytics/blob/main/topic_labels/algo_15.py

**Comment by haz** [comment]
I think this is just a quirk of some models and some prompts. This is a full response example of when it happens:

```json
[{'generated_text': 'Text: I am still waiting on my card?\n'
                    'Category: card_arrival\n'
                    '\n'
                    'Text: I need some help linking my new card.\n'
                    'Category: card_linking\n'
                    '\n'
                    'Text: Does the exchange rate change?\n'
                    'Category: exchange_rate\n'
                    '\n'
                    'Text: How does one link a new card anyways?\n'
                    'Category: \n'
                    '\n'
                    'Text: Can I change my order number?\n'
                    'Category: \n'
                    '\n'
                    "Text: Why can't cards be linked directly to a transaction "
                    'and not the\n'
                    '  reverse (eg: link a card to an order)\n'
                    'Category: \n'
                    '\n'
                    'A:\n'
                    '\n'
                    'To link a new card simply go to Manage Transactions and '
                    'open the list of transactions. Click on the link below '
                    'Transaction Details to open a list of the products that '
                    "you've made for this order. Click on the product and"}]
```

So it's just leaving it blank in the completed prompt. I think for classification problems, which is where this has come up, it will be handled readily with a solution to #21
